packages/vaadin-lumo-styles/components/big-decimal-field.css

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license
+ * Copyright (c) 2017 - 2025 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+@import '../src/mixins/field-base.css';
+@import '../src/mixins/field-button.css';
+@import '../src/mixins/field-error-message.css';
+@import '../src/mixins/field-helper.css';
+@import '../src/mixins/field-label.css';
+@import '../src/mixins/field-required.css';
+@import './input-container.css';
```

Comment:
Could we solve the issue with 

```
get lumoInjector() { 
  return { is: 'vaadin-text-field' } 
}
```

?

Reply:
Seems to work, thanks. I'll close this PR then.